arangod/Cluster/SynchronizeShard.cpp

```diff
@@ -909,15 +855,9 @@ bool SynchronizeShard::first() {
       }
 
       SyncerId syncerId = syncRes.get();
-      // From here on, we have to call `cancelBarrier` in case of errors
-      // as well as in the success case!
-      auto barrierId = sy.get(BARRIER_ID).getNumber<int64_t>();
       NetworkFeature& nf = _feature.server().getFeature<NetworkFeature>();
```

Comment:
It looks like `nf` is not need any further in this function and can also be deleted.